qemu/tests/cfg/blockdev_commit_auto_readonly.cfg

```diff
@@ -0,0 +1,29 @@
+- blockdev_commit_auto_readonly:
+    type = blockdev_commit_auto_readonly
+    virt_test_type = qemu
+    only Linux
+    only filesystem nbd
```

Comment:
we may remove nbd for this case is for filesystem only

Reply:
Remove "only filesystem nbd" to make default "only filesystem". Thanks zhenchao.